.github/workflows/runtests.yml

```
@@ -66,8 +66,25 @@ jobs:
           flags: unittests
           name: PR_tests
           token: ${{ secrets.CODECOV_TOKEN }}
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)